### DISCUSSION TITLE
Sliding a security vulnerability item to read fails

### DISCUSSION THREAD

**Discussion by runely** [community/community#70602] [Mobile]
When sliding or opening a security vulnerability message the following red message appears and the message isn't set as read

![image](https://github.com/community/community/assets/25814673/36bc9554-085c-4298-b1a9-218f8d8768d8)

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.135.0 (112648507)
Device: iPhone15,3 (iOS 17.1)
</details>

**Comment by sskorol** [comment]
Yeah, I believe this bug sits there for a long time. Always postpone to report it.

![image](https://github.com/community/community/assets/6638780/1888b660-64c4-4b7d-be23-e92de0d4188d)
